packages/split-view/README.md

```diff
@@ -0,0 +1,139 @@
+## Description
```

Comment:
We need some sort of intro to the pattern. Sadly, neither CSS or the specification outline a place to start here. How does this maybe sound to start?

> An `<sp-split-view>` element delivers its first two direct child elements in a horizontal or vertical (`<sp-split-view vertical>`) orientation the distributes the available page real estate as per the supplied attribute API. When leveraging the `resizable` attribute a mouse and keyboard accessible affordance is provided to the user to customize the distribution of that area between the available children.

Reply:
Yes, I extended it a bit: 

> An <sp-split-view> element delivers its first two direct child elements in a horizontal or vertical (<sp-split-view vertical>) orientation that distributes the available page real estate as per the supplied attribute API. When leveraging the resizable attribute a pointer and keyboard accessible affordance is provided to the user to customize the distribution of that area between the available children. Providing minimum and maximum panel size values ("primary" and "secondary") disables the collapsing state. This state would allow the user to display one panel at a time completely.

Reply:
I like the information you're adding here, but I'm not sure it's quite clear yet. @najikahalsema as a bit of a third party here, can you share any thoughts you have?

Reply:
Yes, I am also not 100% sure if the last 2 sentences are clear about the intention. So to say if you really *want* to have "collapsed" panels, either remove all min/max values or set min to `0` and max to max-width of the whole component. Or should this be the default and doesn't need any additional explanation?

Reply:
Let's remove `Providing minimum and maximum panel size values ("primary" and "secondary") could disable the collapsing state. This state would allow the user to display one panel at a time completely.` and come back to this in a later PR.

Reply:
Done!